### PR TITLE
Add Broad gnomAD dataset

### PR DESCRIPTION
**Pull request by cseed**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 8597b4fa72e37b28112773739d4f0b4cbe0c8a6d
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=00329ae3-8c8d-4a9a-8142-08011dc444c7%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 7f8b6276d64165e0e0206bd9092a98732728ed08
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=c34f2dcc-3ecd-44c0-b454-ed07919365fd%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: d49b26ffa0087b377c575aefe4151792a3023522
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=11c9571d-3741-473e-922b-bf684ac9cddc%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/broad-gnomad.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      AuthorURL: https://github.com/orgs/broadinstitute/teams/gnomad/members
+    - Title: Hail
+      URL: https://hail.is/
+      AuthorName: Hail Team
+      AuthorURL: https://hail.is/
+    - Title: Hail on AWS Quick Start
+      URL: https://aws.amazon.com/quickstart/architecture/hail/
+      AuthorName: Amazon Web Services and "[PrivoIT](https://www.privoit.com/)"
+  Publications:
+    - Title: 2020 gnomAD Nature Collection
```
Comment (cseed) [inline datasets/broad-gnomad.yaml:43]: The Nature collection is a collection of 7 separate papers. I've asked how the gnomAD team want to specify it, or if we should just break out the 7 papers.

**Comment by cseed** [comment]
@chueatwork I expanded out the gnomAD paper collection.  This should be good to go now.

**Pull request merged**